content/posts/2023-05-15-mac-learning.md

```diff
@@ -0,0 +1,303 @@
+---
+title: "MAC Learning Basics"
+date: 2023-05-15 09:03:20 +0200
+draft: false
+---
+
+This post shows illustrates _dynamic learning of unicast MAC addresses_
+on Linux MAC Bridges, and how to use the `bridge fdb show`
+command to view the MAC database. It expands an [earlier post][Bridging-VLAN-Post] on
+Bridging and VLANs.
+
+
+If you wish to try this out yourself, there are hints on exploring _MAC
+Learning with Infix and GNS3_ at [the
+bottom](#exploring-mac-learning-with-infix) of this post.
+
+**\#learning-cache \#mac-database \#forwarding-database \#fdb
+\#mac-bridge \#transparent-bridge**
+
+
+
+## Example Topology - A single bridge
+
+The figure below shows a sample topology with a bridge (Bridge-A) and
+two connected hosts.  H1 and H2 have IP addresses on the same
+10.0.1.0/24 subnet. The MAC addresses of H1, H2 and Bridge-A have been
+selected for the purpose of easy illustration only. In typical
+scenarios, these are [assigned by the hardware
+manufacturer][wiki-mac].
+
+```
+   Bridge-A BaseMAC
+   02:aa:aa:aa:00:00
+   .----------------------.
+   |        Bridge-A      |
+   | eth0    eth1    eth2 |
+   '--+-------+-------+---'
+      |       |       |
+      |       | 
+	  |     .-+--.
+	  |     | H2 |
+	  |     '----'
+   .--+-.   IP: 10.0.1.2/24
+   | H1 |  MAC: 02:22:22:00:00
+   '----'
+ IP: 10.0.1.1/24
+MAC: 02:11:11:11:00:00
+```
```

Comment:
Mark and `M-x untabify` to avoid any indentation issues.

Reply:
Bra fångat, och jättetack för tipset om M-x untabity!